homepage.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    
+    <div class="container">
+        <div class="toolbar">
+            <div class="logo">
+                <img src="images/debtext.png" alt="LOGO">
+            </div>
+            <div class="space"></div>
+            <button id="menu" class="button">
+                <img src="images/Menu@1x.png" alt="Header">
+            </button>
+        </div>
+
+        <div class="headline">
+            <h1 id="primarytext">Hey There. I'm Debby</h1>
+            <h2 id="supportingtext">I'm a Software Developer</h2>
+            <p id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <div class="socialmedia">
+                <button class="button">
+                    <img src="images/GitHub.png" alt="Github">
+                </button>
+                <button class="button">
+                    <img src="images/Linkedin.png" alt="Linkdin">
+                </button>
+                <button class="button">
+                    <img src="images/Angelist.png" alt="Angelist">
+                </button>
+                <button class="button">
+                    <img src="images/Twitter.png" alt="Twitter">
+                </button>
+                <button class="button">
+                    <img src="images/Medium.png" alt="Medium">
+                </button>
```

Comment:
- [ ] The image list should be wrapped in a `ul` tag, and each `img` should be inside `li`, also the images should be a link so you need to add an `a` tag inside the `li`